test/integration/targets/git/tasks/submodules.yml

```diff
@@ -36,25 +37,55 @@
     state: absent
     path: "{{ checkout_dir }}"
 
+- name: SUBMODULES | Test that clone with non-recursive submodule init retrieves one level of submodules
+  git:
+    repo: "{{ repo_submodules }}"
+    dest: "{{ checkout_dir }}"
+    recursive: no
+    submodule_init: yes
+
+- command: 'ls -1a {{ checkout_dir }}/submodule1'
+  register: submodule1
+
+- assert:
+    that: '{{ submodule1.stdout_lines | length }} == 6'
+
+- command: 'ls -1a {{ checkout_dir }}/submodule1/submodule2'
```

Comment:
Could you please use https://docs.ansible.com/ansible/devel/modules/find_module.html instead of `command`?